index.js

```diff
@@ -114,6 +114,10 @@ class OError extends Error {
 
     if (typeof oError.info === 'object') Object.assign(info, oError.info)
 
+    if (oError.cause) {
+      Object.assign(info, OError.getFullInfo(oError.cause))
```

Comment:
To follow up on the previous discussion about the order of the merge, causes are slightly different from tags because they are lower in the stack. When you tag, you add context higher up the stack, so it makes sense that info from tags supersedes info from the original error. With causes, it's the opposite.